### DISCUSSION TITLE
GitHub Repo is growing very fast - how can it be stopped?

### DISCUSSION THREAD

**Discussion by thomasmerz** [upptime/upptime#247] [Q&A] [answered]
My repo has 700+ MByte in less than two weeks / 14 days! How can this growth be stopped or slowed down? **Cloning and pulling** to my home computer(s) **consumes too much time and too much space for no good reason** 😞  I already lowered the frequency of my CI jobs to every 10 minutes, but this seems not to solve this… 🤔

**Comment by thomasmerz** [comment, marked as answer]
@AnandChowdhary , do you have any idea or tipp for me that I don't know?

**Reply by pcho** [reply]
Clone your repository with —depth option like that ```git clone —depth=1 git@github.com:upptime/upptime.git```.

**Reply by thomasmerz** [reply]
Thanks, @monkinco!  
`git clone --depth=twice-or-triple-of-daily-commits git@github.com:GH_USER/upptime.git` should be a **good compromise between local history/log and local disk usage** 👍🏻

**Reply by thomasmerz** [reply]
I found out that you might want to **use LFS**, so that the upptime repo won't grow too much/too fast. And if it's already "too big" you might want to clone it without BLOBs:
`git clone --filter=blob:none git@github.com:xxx/upptime.git`

@AnandChowdhary , I couldn't find any hint about LFS or blob-less cloning in the docu. Do you want to add this for a better experience when using your brilliant tooling? 🤔
